modules/serverless-creating-broker-admin.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * serverless-using-brokers.adoc
+
+[id="serverless-creating-broker-admin_{context}"]
+= Creating a broker for your namespace automatically using namespace annotation
+
+If you have cluster administrator permissions, you can create a broker automatically by annotating a namespace.
+
+.Prerequisites
+* Knative Eventing installed.
+* Cluster administrator permissions for {product-title}.
+
+.Procedure
+
+. Annotate your namespace (replace `default` with the desired namespace):
```

Comment:
This parenthetical could alternatively also be a callout on the `oc label` line below.